demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -811,8 +825,8 @@ export class DemoMeetingApp
           this.audioVideo.stopVideoPreviewForVideoInput(
             document.getElementById('video-preview') as HTMLVideoElement
           );
+          await this.discardVideoFilterStream();
```

Comment:
Can you put a comment here why we need to have this order?

Reply:
I have added the comment. Good call. Thanks!!

Reply:
I thought we already kill the mediaStream in stopVideoPreviewForVideoInput? I think we only need to stop the transform processor, right?

Reply:
Yes I see your point. I will not try to remove the stream again as that is just useless processing. Thank you.